buildSrc/src/main/groovy/org/elasticsearch/gradle/test/RestTestPlugin.groovy

```diff
@@ -34,6 +34,7 @@ class RestTestPlugin implements Plugin<Project> {
         integTest.configure {
             classpath = project.sourceSets.test.runtimeClasspath
             testClassesDir project.sourceSets.test.output.classesDir
+            cluster.distribution = 'zip' // rest tests should run with the real zip
```

Comment:
this is because the unit test framework is trappy, it supports broken things like `classpath plugins` which is used for testing. the problem there is, no isolation and then you the possibility of people writing messy tests that directly depend on groovy.